cmd/config.go

```diff
@@ -19,7 +18,8 @@ var configurationCmd = &cobra.Command{
 	Long: `Modifies Odo specific configuration settings within the config file.
 
 Available Parameters:
-UpdateNotification - Controls if an update notification is shown or not (true or false)`,
+UpdateNotification - Controls if an update notification is shown or not (true or false)
+NamePrefix - Uses the value of this configurable as a prefix for naming app/component.If set to empty string, the name will be auto generated based on name of component/component source/current directory`,
```

Comment:
Too long. No need for such a long description :(

Maybe shorten to just: `Default prefix is 'odo'. Use this value to set a default name prefix`?

Reply:
No, default prefix is current directory name.
So, changed it to the following:"NamePrefix - Default prefix is current directory name. Use this value to set a default name prefix."

Reply:
Still doesn't show that you updated this comment..

Grammar is also: `Default prefix is the current director name. Use this value to set a default name prefix.`